src/index.html

```diff
@@ -3,10 +3,454 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,500;1,700&display=swap" rel="stylesheet">
+
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <div class="icon">
+        <a
+        href="#head"
+        class="icon__link"
+        ><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="nav nav--position">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple " class="is-active">apple</a></li>
+          <li class="nav__item"><a href="#samsung">samsung</a></li>
+          <li class="nav__item"><a href="#smartphones">smartphones</a></li>
+          <li class="nav__item" data-qa="nav-hover"><a href="#laptops&computers">laptops & computers</a></li>
+          <li class="nav__item"><a href="#gadgets">gadgets</a></li>
+          <li class="nav__item"><a href="#tablets">tablets</a></li>
+          <li class="nav__item"><a href="#photo">photo</a></li>
+          <li class="nav__item"><a href="#video">video</a></li>
```

Comment:
<img width="1087" alt="Screenshot 2023-06-29 at 18 14 50" src="https://github.com/mate-academy/layout_catalog/assets/135232001/0bf675aa-efa7-42eb-a673-f51994da8c40">
Check your code via the checklist from the task